withdraw-by-check/REPWRITERSPECS/BANNO.CHECK.WITHDRAW.V1.POW

```diff
@@ -706,6 +805,25 @@ PROCEDURE READCONFIGFILESETTINGS
            PARAMCUSTOMTERMS(PARAMCUSTOMTERMLINES)=SEGMENT(TMPCHR,1,123)
           END
         END
+
+[* Custom error display messages
+*]
+       ELSE IF SEGMENT(PARAMNAME,1,3)="EDM" THEN
+        DO
+         TMPERRCODE=VALUE(SEGMENT(PARAMNAME,4,6))
```

Comment:
Just a consideration... Instead of looking for specific segment here to get the error code value, maybe consider using the VALUE statement on the full PARAMNAME to eliminate the possibility that the user accidently includes a space, or adds a dash, etc. in between the 'EDM' and the error code.

Reply:
I feel like the the correct format is detailed in the CFG and probably should not be interpreting the param differently than the required/specific format.  
```
Line must be formatted as "EDMxxx:" where xxx represents the 3-digit error code that is returned to the UX.
```
I get what you are saying but how would something like "EDM  5     5FDFD  6". Would that actually be valid in that we would interpret it as error code 556?

Reply:
OK